_includes/sidenav.html

```diff
@@ -50,11 +46,14 @@ <h1 class="u-visually-hidden">
         </button>
     </div>
     <div class="expandable_content">
+        <hr class="inset-divider" />
         <ul class="nav-secondary_list">
         {%- for href, id, caption in items %}
-            <li class="{{ 'nav-secondary_item' if loop.index == 1 else 'nav-secondary_subitem' }}">
+            {# TODO: logic around loop.index needs to be updated
+                     so a parent item can appear before first item. #}
+            <li class="nav-secondary_item{{ ' nav-secondary_item-parent' if loop.index == 1 else ' nav-secondary_item-child' }}">
                 <a class="nav-secondary_link{{ ' nav-secondary_link__current' if id == active_item_id }}"
-                   href="{{ href|e }}">
+                   href="{{ href|e if id != active_item_id }}">
```

Comment:
It should be checked in IE I guess http://stackoverflow.com/questions/5637969/is-an-empty-href-valid
Pushed https://github.com/cfpb/cfgov-refresh/commit/013f13a350adff3eb552466f11aeb6f15e689fea but it can be reverted.